peripheral.joystick/resources/buttonmaps/xml/android/Xbox_360_Wireless_Receiver_v045E_p02A1_23b_6a.xml

```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" ?>
 <buttonmap>
-    <device name="OUYA Game Controller" provider="android" vid="2836" pid="0001" buttoncount="18" axiscount="6">
+    <device name="Xbox 360 Wireless Receiver" provider="android" vid="045E" pid="02A1" buttoncount="23" axiscount="6">
```

Comment:
Is this change intended?

Reply:
wtf Looks like a GIT rename in combination with a file change went wrong when committing the changes :(

Reply:
ok, just checked the individual commits and there it's fine (the ouya git rename still got lost somehow though). Seems to be github confusing things in the "Files changed" view

Reply:
This isn't the fault of GitHub, it's just how git works.

Reply:
@Rechi - if you check the individual commits here on Github then you see that what Garbear is commenting on is not part of any of the changes there.